.github/workflows/stale.yml

```
@@ -0,0 +1,52 @@
+    - cron: '26 17 * * *' # Runs daily at 17:26 UTC
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+
+    steps:
+      - name: Mark stale issues and pull requests
+        uses: actions/stale@v5
```

actionlint (1.7.4)

the runner of "actions/stale@v5" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)